proto/commonfate/control/diagnostic/v1alpha1/diagnostic.proto

```diff
@@ -54,6 +58,27 @@ message ListBackgroundJobsResponse {
   repeated Job jobs = 1;
 }
 
+message ListBackgroundJobKindSummaryRequest {
+  
+}
+
+message JobSummary {
+
+  string id = 1;
+  string kind = 2;
+  string errors = 3;
```

Comment:
Should this be repeated to align with the `Job` type below?
```suggestion
  repeated string errors = 3;
```

or otherwise maybe just `error` - no plural on the name?

Reply:
Looking at the other code I think this would also suit being a slice of errors